docs/build-for-developers/cli-usage.md

```diff
@@ -1,98 +1,160 @@
+**Pass a path to a locally installed adaptor**
 
 ```bash
-openfn help
+openfn path/to/job.js -a http=/repo/openfn/adaptors/my-http-build
 ```
 
-### Run a job
+### Write resulting state to disk
+
+After the job finishes, the CLI writes the resulting state to disk. By default,
+it creates an `output.json` next to the job file. You can specify custom paths
+for the output and state files.
+
+```bash
+openfn path/to/job.js -ia adaptor-name -o path/to/output.json -s path/to/state.json
+```
+
+### Pass state through stdin and return output through stdout
+
+Use `-S` and `-O` to pass state through stdin and return the output through
+stdout.
+
+```bash
+openfn path/to/job.js -ia adaptor-name -S '{"data": "Hello world!"}' -O
+```
+
+### Adjust logging level
+
+You can pass `-l info` or `--log info` to get more feedback about what's
+happening during runtime. Below is the list of different log levels
+
+| log level                                     | description                                              |
+| --------------------------------------------- | -------------------------------------------------------- |
+| `openfn path/to/job.js -a adaptor -l none`    | Quiet mode                                               |
+| `openfn path/to/job.js -a adaptor -l default` | Top level information of what is happening               |
+| `openfn path/to/job.js -a adaptor -l info`    | Get more feedback about runtime, cli and the job         |
+| `openfn path/to/job.js -a adaptor -l debug`   | Get information about runtime, cli, compiler and the job |
+
+### Maintain auto-installed adaptors repo
+
+**List the repo contents**
 
 ```bash
-openfn path/to/job.js -ia {adaptor-name}
+openfn repo list
 ```
 
-Note: You MUST specify which adaptor to use. Pass the `-i` flag to auto-install
-that adaptor (it's safe to do this redundantly).
+**Specify the repo folder using the `OPENFN_REPO_DIR` env var**
 
-You can find the list of publicly available adaptors [here](/adaptors).
+```bash
+export OPENFN_REPO_DIR=/path/to/repo
+```
 
-> Path is the job to load the job from (a .js file or a dir containing a job.js
-> file) For example `openfn execute hello.js ` Reads hello.js, looks for state
-> and output in foo
+**Auto-install adaptors and check if a matching version is found in the repo**
 
 ```bash
--i, --autoinstall Auto-install the language adaptor
--a, --adaptors, --adaptor A language adaptor to use for the job
+openfn path/to/job.js -ia adaptor-name
 ```
 
-If an adaptor is already installed by auto install, you can use the command
-without the `-i` options. i.e `openfn hello.js -a http`
+**Remove all adaptors from the repo**
 
-### Change log level
+```bash
+openfn repo clean
+```
 
-You can pass `-l info` or `--log info` to get more feedback about what's
-happening, or `--log debug` for more details than you could ever use. Below is
-the list of different log levels
+### Run adaptors from monorepo
+
+**Load from the monorepo using the `-m` flag**
```

Comment:
I've lost my context, but have we explained what the monorepo is? Or why this is useful?

Reply:
It's useful for adaptors development, I think it make sense for adaptors documentation not for `cli`. So i will remove this section from here